.github/workflows/ci.yaml

```diff
@@ -147,16 +147,26 @@ jobs:
       REF_PRAXIS: ${{ needs.metadata.outputs.REF_PRAXIS }}
     secrets: inherit
 
-  build-rosetta:
+  build-rosetta-t5x:
     uses: ./.github/workflows/_build_rosetta.yaml
     needs: [metadata, build-t5x]
     with:
       BUILD_DATE: ${{ needs.metadata.outputs.BUILD_DATE }}
       BASE_IMAGE: ${{ needs.build-t5x.outputs.DOCKER_TAGS }}
+      BASE_LIBRARY: t5x
+    secrets: inherit
+
+  build-rosetta-pax:
```

Comment:
We can use [matrix build](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs) here to build Rosetta against t5x/pax.

Reply:
Postponed due to [limitation in GitHub Actions workflow dependency](https://github.com/orgs/community/discussions/16059).